Detectors/MUON/MCH/Calibration/src/BadChannelCalibrationDevice.cxx

```diff
@@ -72,16 +72,26 @@ void BadChannelCalibrationDevice::logStats(size_t dataSize)
 
 void BadChannelCalibrationDevice::run(o2::framework::ProcessingContext& pc)
 {
+  static bool skip = false;
```

Comment:
This might not matter much, but the usage of this static means that the same ECS environment can *not* be reused to perform two pedestal runs, right ?

Reply:
Good point! I have replaced the static variable by a member function, to avoid leaving some herd-to-debug checks in the code.